### ISSUE TITLE
accessibility: accept/decline call buttons cannot be pressed with talkback

### ISSUE THREAD

**Issue by beqabeqa473** [BelledonneCommunications/linphone-android#193]
hello all.

i am using linphone and have an issue.

i cannot answer/decline incomming calls with using talkback screenreader with double tap on these buttons.

what can be done to improve the situation?

**Comment by schulle4u** [comment]
I can confirm this behaviour on my device (Motorola Moto G4 Plus, Android 7.0). Just installed Linphone from F-Droid and found it to be very accessible for TalkBack users, except for call answer and decline buttons. These cannot be executed using normal double tap gesture with TalkBack. Currently the only workaround is to use auto answering feature, which of course isn't wanted in every situation.

**Comment by schulle4u** [comment]
Just opened another issue on the belledonne Gitlab, seems this repo only is a mirror. 
https://gitlab.linphone.org/BC/public/linphone-android/issues/1

**Comment by Viish** [comment]
Let's continue this conversation on our gitlab
